README.md

```diff
@@ -15,14 +15,21 @@ Installing the module:
 
 # Settings
 ## Avalara connection settings
-The module can be configured in two places:
+The module can be configured in three places:
+* Platfrom config file: appsettings.json
 * Platform-wide settings: Settings -> Taxes -> Avalara or Modules -> Installed -> Avalara tax -> Settings
 * Store-specific settings: Stores -> (your store) -> Tax providers -> Avalara taxes
 
-Both of these places have the following settings:
-* **Avalara.Tax.IsEnabled** - Enable or disable tax calculation
-* **Avalara.Tax.Credentials.AccountNumber** - Account number provided by Avalara during registration process
-* **Avalara.Tax.Credentials.LicenseKey** - Account License Key provided by Avalara during registration process
+**AccountNumber** and **LicenseKey** provided by Avalara during registration process should be configured in appsetting.json:
```

Comment:
**AccountNumber** and **LicenseKey** provided …
==>
Confidential account settings - **AccountNumber** and **LicenseKey** - provided …

Reply:
fixed